src/ConfigurationProvider.php

```diff
@@ -76,7 +76,7 @@ public function register(Application $app)
      */
     public static function isDev()
     {
-        return (new RuntimeSettings($_SERVER))->isDev();
+        return (new RuntimeSettings($_SERVER, []))->isDev();
```

Comment:
It used to be static so it can be used before the DI version is initialized. And actually it needs $_GET, I'll add it back.